companies/monki.json

```diff
@@ -0,0 +1,37 @@
+{
+    "slug": "monki",
+    "relevant-countries": [
+        "at",
+        "be",
+        "cz",
+        "dk",
+        "fi",
+        "fr",
+        "de",
+        "hu",
+        "ie",
+        "it",
+        "nl",
+        "pl",
+        "pt",
+        "sk",
+        "sl",
+        "es",
+        "se",
+        "gb"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Monki H&M Hennes & Mauritz Sverige AB",
```

Comment:
Since this is apparently also H&M, why is it not part of the `runs` array? Just curious.

Reply:
That is also a case-by-case decision for me. As for the H&M ones, my thinking here is that they all a) 'want to appear' (I genuinely don't think there is a good translation for 'auftreten' in that context) as separate companies and b) they have separate contact details for each of the websites, so it's probably best for a user to contact the website they actually interacted with as opposed to one of the others who would probably just direct them there anyway.

The `runs` is more for companies like Atlassian, Automattic or myToys, where they offer separate websites/services (sometimes even companies) but there is clearly a single responsible (in terms of responding to privacy-related requests) entity behind them that can be reached using a single set of contact details.

That is also something that the alternative schema described above couldn't account for (and I don't think there even is a reasonable way to do so), so we will likely have to stick with separate records.